### PR TITLE
Add Prime95 v28.9

### PR DESCRIPTION
**Pull request by farski**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/prime95.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'prime95' do
+  version '28.9'
+  sha256 'c8e58f032dbbf0402d97ea81272153faeae61d6ef4461c1f88c99ecbf3fdcc32'
+
+  url 'http://www.mersenne.org/ftp_root/gimps/p95v289.MacOSX.zip'
```
Comment (vitorgalvao) [inline Casks/prime95.rb]: `url "http://www.mersenne.org/ftp_root/gimps/p95v#{version.no_dots}.MacOSX.zip"`

File: Casks/prime95.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name 'Prime95'
+  homepage 'http://www.mersenne.org/download/'
```
Comment (vitorgalvao) [inline Casks/prime95.rb]: `homepage 'http://www.mersenne.org/'`

File: Casks/prime95.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'prime95' do
```
Comment (vitorgalvao) [inline Casks/prime95.rb]: Needs [a comment](https://github.com/caskroom/homebrew-cask/blob/5b19785263e6092a8198b1c7fb63e7b4a94f1168/Casks/p5.rb#L2).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

**Comment by farski** [comment]
@vitorgalvao I think I've corrected all the issues, sorry about that!

**Comment by vitorgalvao** [comment]
You did. Thank you.

**Pull request merged**